nav2_collision_monitor/src/collision_detector_node.cpp

```diff
@@ -259,26 +259,33 @@ bool CollisionDetector::configureSources(
         rclcpp::ParameterValue("scan"));  // Laser scanner by default
       const std::string source_type = get_parameter(source_name + ".type").as_string();
 
+      nav2_util::declare_parameter_if_not_declared(
+        node, source_name + ".source_timeout",
+        rclcpp::ParameterValue(source_timeout.seconds()));    // node source_timeout by default
+      const rclcpp::Duration sensor_specific_source_timeout = rclcpp::Duration::from_seconds(
+        get_parameter(
+          source_name + ".source_timeout").as_double());
```

Comment:
`source_name + ".source_timeout").as_double());` can be on the same line as `get_parameter`, looks like an autoformatting oddity

Reply:
^ I wouldn't have bothered if I didn't ask for the log update below

Reply:
Fixed